Answers.md

```diff
@@ -1,11 +1,12 @@
 # Answers
 
 1. What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
-
+it uses JSX, makes sites faster
 1. What does it mean to think in react?
-
+To think in react means to think in the operation of a single page website. what that means is is to think fast and operative. 

```

Comment:
As you have rightly stated, React makes the creation of fast and operative single page applications easier.
However, thinking in React in this context is referring to composing React components. [This is an excellent article to that effect](https://reactjs.org/docs/thinking-in-react.html).